docs/OncoTree-Mapping-Tool.md

```diff
@@ -184,3 +188,26 @@ After filling in the `ONCOTREE_CODE` column with an OncoTree code for each sampl
 |P16|S16|65|MNM|
 |P17|S17|66|MYCF|
 |P18|S18|66|RBL|
+
+# Ontology Mapping Tool
+
+The Ontology Mapping tool was developed to facilitate the mapping between different cancer classification systems. We currently allow the mappings between OncoTree, ICD-O, NCIt, UMLS and HemeOnc systems. The tool and the mapping file can be found [here](https://github.com/cBioPortal/oncotree/tree/master/scripts)
+
+### Running the tool
+
+The tool can be run with the following command:
+```
+python <path/to/scripts/ontology_mapping_tool.py> --source-file <path/to/source/file> --target-file <path/to/target/file> --target-code <target_ontology_code>
+```
+
+**Options**
+```
+ -i | --source-file: This is the source file path. The source file must contain one of the ONCOTREE_CODE, ICDO_CODE, NCIT_CODE, UMLS_CODE or HEMEONC_CODE in the file header and it must contain codes corresponding to the Ontology System.
+ -o | --target-file: This is the path to the target file that will be generated. It will contain ontologies mapped from source code in <source-file> to <target-code>.
+ -t | --target-code: This is the target ontology codes that the script will attempt to map the source file ontology codes to. It must be one of the ONCOTREE_CODE, ICDO_CODE, NCIT_CODE, UMLS_CODE or HEMEONC_CODE.
+```
+
+**Note:**
+- The Ontology Mapping tool runs on python 3.
+- The source file should be tab delimited with one or more columns and it should contain atleast one of ONCOTREE_CODE, ICDO_CODE, NCIT_CODE, UMLS_CODE or HEMEONC_CODE in the file header.
+- We currently are allowing only one ontology to another ontology mapping. This tool will be extended to support one to multiple ontology systems mapping.
```

Comment:
I would be a little less definite about extending the tool to support multiple mappings (unless we have a delivery date in mind). So I would suggest something like "In the future, we hope to extend the tool to support mapping to multiple ontology systems", or "We plan to extend the tool to support mapping to multiple ontology systems."
Also ... wouldn't you be able to run the tool 3 times in order to add three external ontologies? Maybe it is not so important to let the user do all three in one step? On the other hand, maybe it would not be too difficult to extend it so that it would do 3 at once.

Reply:
Agreed and updated per your suggestion.